draft.md

```diff
@@ -53,7 +53,8 @@ This week's release was curated by [](), with help from the R Weekly team member
 ### Videos and Podcasts
 
 + [Listen to the R-Weekly Highlights Podcast](https://rweekly.fireside.fm/)
-
++ [How to fix the sizes in your ggplots](https://youtu.be/7kWW8rkpfqc)
```

Comment:
Hi @AlbertRapp! The plot sizes vid was already included in this (past) weeks edition. Happy to accept the other one though.

Reply:
Oh my. Total blunder on my part. I thought I didn't submit it yet. Thanks for watching out!

Reply:
Now I only have to figure out how to make the change in a commit. Or is this sth only you can do on your part?